streams/src/test/java/org/apache/kafka/streams/state/internals/CachingSessionStoreTest.java

```diff
@@ -49,14 +51,15 @@
 import static org.junit.Assert.assertFalse;
 
 
+@SuppressWarnings("PointlessArithmeticExpression")
 public class CachingSessionStoreTest {
 
     private static final int MAX_CACHE_SIZE_BYTES = 600;
-    private InternalMockProcessorContext context;
+    private static final Long DEFAULT_TIMESTAMP = 10L;
+    private static final long SEGMENT_INTERVAL = 100L;
```

Comment:
I searched through the code base to find all the spots where we set the segment interval to 1 minute just because it was the min. I've set them all lower (typically to 100ms, because it's a nice round number) mostly to avoid implying that there's anything special about 60s anymore.